develop/tutorials/articles/110-portlets/03-spring-mvc/00-spring-mvc-intro.markdown

```diff
@@ -58,9 +58,10 @@ Here's an example `Import-Package` header:
         org.springframework.web.servlet.config.MvcNamespaceHandler
 
 The auto-deploy process and Liferay's WAB generator convert your project to a
-Liferay-ready WAB. The WAB generator producees the WAB manifest's
-`Import-Package` header by merging `liferay-plugin-package.properties`'s
-packages with the packages your portlet's classes import.
+Liferay-ready WAB. The WAB generator produces the WAB manifest's
+`Import-Package` header by merging the packages declared in your plugin's
+`liferay-plugin-package.properties` file with the packages that contain your
+project's classes.
```

Comment:
I don't think "packages that contain your project's classes" is accurate. Isn't it that the packages your classes import must be added to `Import-Package`?

Reply:
I was thinking along the lines that classes don't import packages, modules do.
Classes have import statements but those are just shortcuts so the fully qualified class names don't have to be used.